src/strands/multiagent/swarm.py

```diff
@@ -303,15 +336,32 @@ async def invoke_async(
                     self.execution_timeout,
                 )
 
-                await self._execute_swarm(invocation_state)
+                async for event in self._execute_swarm(invocation_state):
+                    yield event
+
+                # Yield final result (consistent with Agent's AgentResultEvent format)
+                result = self._build_result()
+                yield {"result": result}
```

Comment:
> Yield final result (consistent with Agent's AgentResultEvent format)

Is this an AgentResult as it is for [single-agent streaming](https://strandsagents.com/latest/documentation/docs/user-guide/concepts/streaming/overview/#lifecycle-events)?  If not, can we rename so that it doesn't conflict with different types